ios/BT/API/Requests/DiagnosisKeyRequests.swift

```diff
@@ -60,8 +60,18 @@ enum DiagnosisKeyListRequest: APIRequest {
         "appPackageName": Bundle.main.bundleIdentifier!,
         "verificationPayload": certificate,
         "hmackey": hmacKey,
-        "padding": String(decoding: Data(), as: UTF8.self)
+        "padding": Data.randomPadding(size: 1000)
```

Comment:
can you use a constant for that 1000 and maybe a short description of why that number?